gp-limit-choices/gplc-spots-left-plus-waiting-list.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Gravity Perks // GP Limit Choices // Spots Left + Waiting List Message
+ *
+ * This snippet allows you to display the number of spots left in the label of each choice. If there are no spots left, it will display a waiting list message.
+ * https://gravitywiz.com/documentation/gp-limit-choices/
+ *
+ * Plugin Name:  GP Limit Choices — Spots Left + Waiting List Message
+ * Plugin URI:   https://gravitywiz.com/documentation/gp-limit-choices/
```

Comment:
Let's update this URL to "https://gravitywiz.com/documentation/gravity-forms-limit-choices/"